### DISCUSSION TITLE
Several criteria in Applicability. AND or OR?

### DISCUSSION THREAD

**Discussion by Xema-Pastor** [buildingSMART/IDS#360] [Q&A] [answered]
Hello everybody.

If you want to sepecify an applicability filter involving two or more criteria, how does you differentiate an AND and an OR.

I have noticed that an applicability filter specified this way:
```
The model must contain entities that have:
- an attribute ObjectType
- an attribute Representation
```

are understood as "ObjectType AND Representations" in some checker engines, and "ObjectType OR Representations" in others.

Which is the correct one?

_(I would like to specify an OR filter criteria)_

Thanks

**Comment by NickNisbet** [comment]
This is the difference between an Applicability (joined by AND) and a Selection ( joined by OR).

Laugh? No, I nearly cried.

**Comment by atomczak** [comment, marked as answer]
Hi @Xema-Pastor, applicability facets are joint with AND. To achieve OR in your case, use enumeration:

```      <ids:applicability minOccurs="1" maxOccurs="unbounded">
        <ids:attribute>
          <ids:name>
            <xs:restriction base="xs:string">
              <xs:enumeration value="ObjectType" />
              <xs:enumeration value="Representation" />
            </xs:restriction>
          </ids:name>
        </ids:attribute>
      </ids:applicability>
 ```

**Reply by Xema-Pastor** [reply]
Understood.

Thank you so much, @atomczak
